app-editors/nano/nano-3.1.recipe

```diff
@@ -27,12 +27,19 @@ PROVIDES="
 REQUIRES="
 	haiku
 	cmd:groff
+	lib:libintl
+	lib:libmagic
```

Comment:
Oops, I did the same mistake (adding `lib:libmagic`) a few months ago in 4fc82ec8f9616944234701a3b45b414dcaebdded :smile:
Like you, I thought it was a good idea but it was not, as @waddlesplash explained in https://github.com/haikuports/haikuports/pull/2697#issuecomment-397898548. (He fixed my mistake just after.)
(I guess it's still not a good idea, but something might have changed. Well, probably not.)

Regarding the removal of the old recipes, I just realized we had `2.9.8` and 2 older recipes, `2.7.5` and `2.8.7`.
I should have suggested that you could, maybe, only remove one of the older recipes.
But that's ok. If nano `3.1` is working fine then we can drop all the old recipes.

Reply:
>  Like you, I thought it was a good idea but it was not, as @waddlesplash explained in #2697 (comment). (He fixed my mistake just after.)

The code in nano which uses libmagic is about 5 lines long IIRC; someone could just take the time to write a patch that fetches the mimetype using Haiku's native APIs instead.